src/l500/l500-device.cpp

```diff
@@ -619,6 +619,51 @@ namespace librealsense
         return command{ ivcam2::FRB, 0x0011E000, 0x3f8 };
     }
 
+    bool l500_device::is_special_command( const std::vector< uint8_t > & input )
+    {
+        std::string command_str( input.begin(), input.end() );
+
+        if( command_str == "GET-NEST" )
+        {
+            // Handle extended temperature command
+            auto nest_response = _hw_monitor->send( command{ ivcam2::TEMPERATURES_GET } );
+            if( nest_response.size() != sizeof( extended_temperatures ) )
+            {
+                throw invalid_value_exception( "Extended temperatures get FW command failed" );
+            }
+
+            auto const & ext_temp
+                = *( reinterpret_cast< extended_temperatures * >( nest_response.data() ) );
+            LOG_INFO( "Nest AVG: " << ext_temp.nest_avg );
+
+            // Handle other commands (all results log the first byte)
+            log_FW_response_first_byte( "Gain trim",
+                             command( ivcam2::IRB, 0x6C, 0x2, 0x1 ),
+                             sizeof( uint8_t ) );
+            log_FW_response_first_byte( "IPF gain",
+                             command( ivcam2::MRD, 0xA003007C, 0xA0030080 ),
+                             sizeof( uint32_t ) );
+            log_FW_response_first_byte( "APB VBR",
+                             command( ivcam2::AMCGET, 0x4, 0x0, 0x0 ),
+                             sizeof( uint32_t ) );
+
+            return true;
+        }
+        return false;
```

Comment:
You don't want to call the base class?? What if it implements other commands?

Reply:
Not relevant, as discussed